app/admin/lookup/route.js

```diff
@@ -12,10 +12,6 @@ export default AbstractIndexRoute.extend({
     return this.store.findAll('lookup').catch((error) => this.send('error', error));
   },
 
-  afterModel(model) {
```

Comment:
em-select can choose  the first element automatically now, but I did that only here when I realised how big this PR would be